lib/honeybadger.ex

```diff
@@ -286,10 +287,14 @@ defmodule Honeybadger do
       |> contextual_metadata()
       |> Map.put(:breadcrumbs, breadcrumbs)
 
-    exception
-    |> Notice.new(metadata_with_breadcrumbs, stacktrace, fingerprint)
-    |> put_notice_fingerprint()
-    |> Client.send_notice()
+    notice =
+      exception
+      |> Notice.new(metadata_with_breadcrumbs, stacktrace, fingerprint)
+      |> put_notice_fingerprint()
+
+    exclude_error_module = Application.get_env(:honeybadger, :exclude_errors)
+
+    if not exclude_error_module.exclude_error?(notice), do: Client.send_notice(notice)
```

Comment:
```suggestion
    unless exclude_error_module.exclude_error?(notice), do: Client.send_notice(notice)
```
Instead of negating within an `if .. else`, I think is better to use `unless`. What do you think about this?